### ISSUE TITLE
Copy/paste loadout sections

### ISSUE THREAD

**Issue by bhollis** [DestinyItemManager/DIM#9811]
As suggested on Discord, it'd be nice to be able to copy a section (armor, fashion, mods, subclass, etc) from one loadout to another. A couple ways we could approach this:

1. Have a "copy from..." button in the dropdown, and then show a prompt where you can select a loadout to copy from.
2. Use the Clipboard API to allow "Copy Section" and "Paste Section" commands that work between loadouts.

**Comment by bhollis** [comment]
Makes sense. I'd still like feedback from folks on whether they prefer the first or second option for how this is presented to users.

**Comment by jbccollins** [comment]
I think the mod copying might take some thought. slot-agnostic stat mods (like minor resil and recovery forged) would likely not make much sense to copy over between many loadouts as that would mess with the stats that the user wants to achieve. But if you don't copy stat mods, then there might not be a valid placement for the slot-specific mods if receiving loadout uses more expensive slot-agnostic stat mods. 🤔
